src/snapred/backend/dao/normalization/NormalizationRecord.py

```diff
@@ -31,7 +31,7 @@ class NormalizationRecord(Record, extra="ignore"):
     smoothingParameter: float
     # detectorPeaks: List[DetectorPeak] # TODO: need to save this for reference during reduction
     workspaceNames: List[WorkspaceName] = []
-    calibrationVersionUsed: int = VERSION_DEFAULT
+    calibrationVersionUsed: Version = VERSION_START
```

Comment:
A _little_ bit confusing, as it's not obvious then that this means the default version.  Here I'm wondering if we should just _define_ `VERSION_DEFAULT = VERSION_START`?